### ISSUE TITLE
Nextcloud 10 not working

### ISSUE THREAD

**Issue by mhzawadi** [Nethar/owncloud-bookmarks#1]
Hi,

This is a great app and I love using it, but it doesnt work with nextcloud 10. Do you maybe have any ETA for fixing it?

**Comment by Nethar** [comment]
Hi. I don't have time to maintain it with 2 small children, but I opened
the code to GIT and someone is working on it. Hopefully it will work soon.

Dne 20. 11. 2016 17:38 napsal uživatel "Matthew Horwood" <
notifications@github.com>:

**Comment by mhzawadi** [comment]
Thanks for the reply, I can relate to that.
Do you know who is working on it? dont worry if not

**Comment by Nethar** [comment]
Oh, this is already github discussion, I did not realize that when I answered from mail ;-) User https://github.com/theScrabi will probably try to maintain it, but you are free to discuss it with him. Just note that code is not very well written (my one and only Android app with lot of copy-paste). Not sure how much of the code is really reusable with current owncloud login changes and planned bookmark app API changes (well "API", it was more of hacking then using API).
